daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -19,7 +19,7 @@ import com.daml.lf.iface
 import com.daml.lf.iface.EnvironmentInterface
 import com.daml.lf.iface.reader.InterfaceReader
 import com.daml.lf.language.Ast
-import com.daml.lf.language.Ast._
+import com.daml.lf.language.Ast.{Exception => _, _}
```

Comment:
What does this do?

Reply:
Otherwise `Exception` from `com.daml.lf.language.Ast` hides `java.lang.Exception` used line 538. 
Maybe I should use another name to avoid this clash.  `ExceptionDef` ?

Reply:
In the protobuf it's called `DefException`. Maybe that?

Reply:
:+1: